apps/files/src/components/FileEntryMixin.ts

```diff
@@ -208,6 +211,11 @@ export default defineComponent({
 		},
 
 		execDefaultAction(event) {
+			// Ignore right click.
+			if (event.button === 2) {
```

Comment:
Should we ignore all non-main buttons? I mean if I press my tertiary button I would not expect the action to be performed.
The only button I am not sure about is middle-mouse button for opening in new tab?

```suggestion
			if (event.button != 2) {
```

Reply:
From what I understand,
0 is left
1 is middle
2 is right
3 and above is whatever

`if (event.button >= 1) {` ?

Reply:
Makes sense